### PR TITLE
Add Banner component as a separate story and have CookieConsent depend on it

### PR DESCRIPTION
**Pull request by af-afk**

### CODE REVIEW

File: web/app.fluidity.money/app/root.tsx [hunk 1]
```diff
@@ -235,15 +237,41 @@ function App() {
...
         <Meta />
         <Links />
       </head>
       <body>
+        <Banner activated={showTargetedBanner}>
+          <p>Hey there...</p>
+          <h2>Thank you for using the app!</h2>
+          <p>
+            We'd love to sign *YOU* up to our beta program, could you visit
+            <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">this link?</a>
```
Comment (glo12345) [inline web/app.fluidity.money/app/root.tsx:276]: might want to change this

**Review by glo12345 — CHANGES_REQUESTED** [review]

File: web/app.fluidity.money/app/root.tsx [hunk 2]
```diff
@@ -235,15 +237,41 @@ function App() {
...
+  const [showTargetedBanner, setShowTargetedBanner] = useState(false);
+
+  useEffect(() => {
+    if (!showExperiment("target-banner")) {
+      return;
+    }
+
+    const _targetedBannerSeen = localStorage.getItem("targetedBanner");
+    if (!_targetedBannerSeen) {
+      setShowTargetedBanner(false);
```
Comment (glo12345) [inline web/app.fluidity.money/app/root.tsx]: assuming this should be `setShowTargetedBanner(true)`? ie. If they have not seen targetedBanner, then show banner

File: web/app.fluidity.money/app/root.tsx (hunk 1 above)
Comment (af-afk) [reply web/app.fluidity.money/app/root.tsx:276]: No joke, hasn't been defined what it should have yet

File: web/app.fluidity.money/app/root.tsx (hunk 2 above)
Comment (af-afk) [reply web/app.fluidity.money/app/root.tsx]: True, and it should be a lot cleaner too

**Review by glo12345 — APPROVED** [review]

**Pull request merged**